src/hzb_combinatorial_libraries/schema.py

```diff
@@ -752,4 +783,92 @@ def normalize(self, archive, logger: BoundLogger) -> None:
         super(UnoldThermalEvaporation, self).normalize(archive, logger)
 
 
+class PixelProperty(EntryData):
```

Comment:
is there a reason that thickness is not in properties?

Reply:
what does @Pepe-Marquez think?

Reply:
Since we are limiting and focusing ourselves only on the absorber layer, it can go in the properties subsection.